### ISSUE TITLE
Test pulling ACF images from MANN to CULU.

### ISSUE THREAD

**Issue by manolobevia** [cul-it/uls#885]

**Comment by cappadona** [comment]
@manolobevia Maybe try a more simple baseline of standard post with images (no ACF and no CPT)?

**Comment by manolobevia** [comment]
@cappadona Created a [post](https://dev-uls-mann-library-cornell-edu.pantheonsite.io/test-distributor-featured-image/) on Mann unit with a featured image and pulled the post on CULU instance with the featured image from MANN post. You can check the post here at https://dev-culu-library-cornell-edu.pantheonsite.io/wp-admin/post.php?post=1051&action=edit

Also, I have created a [custom post](https://dev-uls-mann-library-cornell-edu.pantheonsite.io/wp/wp-admin/post.php?post=2151&action=edit) with a featured image in Mann, and I was able to pulled the [custom post](https://dev-culu-library-cornell-edu.pantheonsite.io/wp-admin/post.php?post=1053&action=edit) with the featured image

In conclusion, distributor is capable of pulling featured images from a post type post and custom posts. Images are not pulled when used with  ACF `image` or `image aspect ratio crop field` types.

**Comment by cappadona** [comment]
Thanks @manolobevia. This aligns with the [known caveats](https://github.com/10up/distributor#known-caveatsissues) from the distributor README:

> **Post Meta Associations** - A distributed post includes all of the post meta from the original version. Sometimes arbitrary post meta references an ID for another piece of content on the original site. Distributor does not "bring along" the referenced content or update references for arbitrary post meta (it will take care of updating references in the case of core WordPress features, such as the featured image ID). This issue is very common when using field management plugins like Advanced Custom Fields (ACF). This can be addressed on a case by case basis by extending the plugin; for external connections, you can manually handle post meta associations using the dt_push_post hook. For internal connections, use the dt_push_post hook. Note that while named the same, these hooks accept different parameters.

So for now, we will need to use the provided hooks to ensure we associate the appropriate ACF image fields for each of our CPTs that will be distributed.
